packages/assets/scss/03-organisms/_sections-three-up.scss

```diff
@@ -97,4 +107,16 @@ $bp-section-3up-large-max: "max-width:950px";
     }
   }
 
+  .ma__card__wrapper {
```

Comment:
We are trying to componentize the styles as much as we can, so this is considered antipattern and making it hard to debug in the future. Instead, could you add a card property/variant `card.theme = "c-white"` to the card scss and twig file that encapsulate the white card them. And pass that to the org section enhanced in openmass? technically, we'd like to avoid referencing openmass classes in mayflower if possible. Thanks!